frontend/device/gesturedetector.lua

```diff
@@ -371,10 +374,10 @@ Attempts to figure out which clock source tap events are using...
 function GestureDetector:probeClockSource(timev)
     -- We'll check if that timestamp is +/- 2.5s away from the three potential clock sources supported by evdev.
     -- We have bigger issues than this if we're parsing events more than 3s late ;).
-    local threshold = TimeVal:new{ sec = 2, usec = 500000 }
+    local threshold = time.s(2) + time.us(500000)
```

Comment:
(time.s(2.5), but I guess you keep that for later :)

Reply:
No, that's important. (Here 2.5 is exactly encoded in a float) If you want exactly 2.1s you would have to write `time.s(2) + time.us(500000)` or `time.s(2) + time.ms(500)`.  If something very near to 2.1s you can use `time.s(2.1)`.

That's the only drawback of this PR (IMHO).